src/sync/installation.ts

```diff
@@ -178,6 +178,28 @@ const isBlocked = async (installationId: number): Promise<boolean> => {
 	}
 }
 
+// Checks if parsed error type is NOT_FOUND / status is 404 which come from 2 different sources
+// - GraphqlError: https://github.com/octokit/graphql.js/tree/master#errors
+// - RequestError: https://github.com/octokit/request.js/blob/5cef43ea4008728139686b6e542a62df28bb112a/src/fetch-wrapper.ts#L77
+export const handleNotFoundErrors = (
+	err: any,
+	job: any,
+	nextTask: Task
+): boolean | undefined => {
+	const isNotFoundErrorType =
+		err?.errors && err.errors?.filter((error) => error.type === "NOT_FOUND");
+
+	const isNotFoundError = isNotFoundErrorType?.length > 0 || err?.status === 404;
```

Comment:
same

Reply:
The ? checks if these values exist. I have tests that pass when a null value/string/empty string is passed instead of the error object 😄